exporter/api.py

```diff
@@ -13,6 +14,25 @@
 from django.utils.text import slugify
 from django.core.files.storage import default_storage
 from lore.settings import EXPORT_PATH_PREFIX
+from learningresources.models import STATIC_ASSET_BASEPATH
+
+
+def _find_unused_path(path):
+    """Iterate through filenames until we get to one that isn't taken."""
+    base, ext = os.path.splitext(path)
+
+    collision = False
+    i = 1
+    while os.path.exists(path):
+        collision = True
+        path = "{base}_{i}{ext}".format(
+            base=base,
+            ext=ext,
+            i=i
+        )
+        i += 1
+
+    return path, collision
```

Comment:
Just a question: Why is ``collision``  tracked?  It seems to be only used in the tests, and there it is asserted to be ``True``.